docs/overview/about.md

```diff
@@ -87,35 +92,35 @@ Archway seeks to provide three potential funding sources for dApps:
 - Share of inflationary rewards
 - Smart contract fees (optional)
 
-Archway allows each dApp developer and their community to configure how the rewards are managed and distributed. When a contract is instantiated, the dApp creator specifies an `owner` address where all fees and rewards are automatically deposited. This target address can be their own, a multisig, or controlled by a separate custom contract. Ownership can then be transferred to a new address as needed, only requiring the signature of the previous owner. 
+Archway allows each dApp developer and their community to configure how the rewards are managed and distributed. When a contract is instantiated, the dApp creator specifies an `owner` address where all fees and rewards are automatically deposited. This target address can be the creator's address, a multisig, or an address that is controlled by a separate custom contract. Ownership can then be transferred to a new address as needed. Ownership transfer requires only the signature of the previous owner. 
 
 <!-- Economics : Gas Rebates -->
 
 ### Gas Rebates 
 
 Unlike existing smart contract platforms, Archway does not burn gas fees or distribute them entirely to the validators or miners. Instead, the collected gas fees are split between dApp developers and validators.
 
-At network launch, gas fees will be divided evenly with 50% going to dApp developers and 50% to validators. The network provides configurable parameters that can be adjusted over time to determine the optimal gas rebate distribution between validators and dApps.
+At network launch, gas fees are divided evenly with 50% going to dApp developers and 50% to validators. The network provides configurable parameters that can be adjusted over time to determine the optimal gas rebate distribution between validators and dApps.
```

Comment:
even for projects that haven't launched yet, using present tense is the best guidance
https://developers.google.com/style/tense